packages/gateway/src/routes/getMinioDocument/export-route.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+
+import { getMinioDocumentHandler } from '@gateway/routes/getMinioDocument/handler'
+
+export default {
+  method: 'GET',
+  path: '/document',
+  handler: getMinioDocumentHandler,
```

Comment:
Should we call it just a `getDocumentHandler` here and hide the "minio" term inside the handler?